qml/components-felgo/TimePickerDialog.qml

```diff
@@ -0,0 +1,40 @@
+import Felgo 3.0
```

Comment:
Is this from `/showcase/qml/controls/TimePickerDialog.qml`?

Reply:
Yes, sir! See: https://github.com/micuintus/harbour-Berlin-Vegan/pull/331/commits/55ea24d686025d4cd88a5cd66b8e6a4e862a559a